lib/miq_automation_engine/engine/miq_ae_workspace.rb

```diff
@@ -63,23 +59,11 @@ def readonly?
       @readonly
     end
 
-    def self.instantiate_readonly(uri, workspace = nil, root = nil)
-      workspace ||= MiqAeWorkspaceRuntime.new(:readonly => true)
-      self._instantiate(uri, workspace, root)
-    end
-
-    def self.instantiate(uri, workspace = nil, root = nil)
-      workspace ||= MiqAeWorkspaceRuntime.new
```

Comment:
There was only 1 caller. but it looked like it was passed in only to because they wanted a reference. Since the method returns the workspace in the end, it looked unnecessary.